app/builders/v2/report_builder.rb

```diff
@@ -15,7 +15,13 @@ def initialize(account, params)
   end
 
   def timeseries
-    send(params[:metric])
+    metric_name = params[:metric]
+    if metric_name
```

Comment:
If `metric_name` is an empty string, this will evaluate to `true`, in case we wanna avoid that let's use `.present?`

```suggestion
    if metric_name.present?
```